lte/gateway/c/core/oai/test/spgw_task/spgw_test_util.cpp

```diff
@@ -383,5 +385,16 @@ void fill_nw_initiated_deactivate_bearer_response(
   nw_deactv_bearer_resp->s_gw_teid_s11_s4 = sgw_s11_context_teid;
 }
 
+void fill_s11_suspend_notification(
+    itti_s11_suspend_notification_t* suspend_notif, teid_t sgw_s11_context_teid,
+    std::string imsi_str, ebi_t link_bearer_id) {
```

Comment:
```suggestion
    const std::string& imsi_str, ebi_t link_bearer_id) {
```

Reply:
Fixed in https://github.com/magma/magma/pull/10674/commits/bd56937b288b789ea026b02270ea5cbff5189279